agent/util-scripts/gold/test-start-stop-tool-meister/test-51.txt

```diff
@@ -254,5 +254,7 @@ testhost.example.com 0025 INFO pbench-tool-meister __exit__ -- testhost.example.
 testhost.example.com 0026 DEBUG pbench-tool-meister _send_client_status -- publish pbench-agent-cli-from-tms {"hostname": "testhost.example.com", "kind": "tm", "status": "terminated"}
 --- mock-run/tm/tm.logs file contents
 +++ test-execution.log file contents
+/var/tmp/pbench-test-utils/opt/pbench-agent/unittest-scripts/require-rpm pbench-sysstat 12.0.3
+/var/tmp/pbench-test-utils/opt/pbench-agent/unittest-scripts/require-rpm perf 
```

Comment:
We don't care about the version of `perf` because they don't change the interface to its behavior at all.  But for `sysstat`, they have been so bad about compatibility with their output format that we have locked it to 12.0.3 to avoid breaking across different versions, causing us a support night-mare.

This is the reason why we are adding proper support for PCP and Prometheus.

Reply:
Re `sysstat`, locked is fine...but, you don't want to make that relationship explicit by adding an `eq`, now that you can?

Re `perf`, yay them!  However, at the moment I'm in a mood to state our intentions explicitly, so I would put in a plug here for adding `1.0.0 ge` or similar.  :-)

Reply:
We default to `eq`.

I was hoping not to change the `pbench-agent-default.cfg` config file to have that for the tools in this commit.  This will open a can of worms for fixing the way tools are specified.

Reply:
Defaulting to `eq` when the version is specified is fine.  Defaulting to "any" (e.g., `0.0.0 ge`) when the version is omitted should solve rest of the problem.